.idea/jsLibraryMappings.xml

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@megelismi Like Andre said, we should remove this file since it's so large. It may or may not be on the .gitignore file already, but you're going to have to manually remove it using:

`git rm --cached -r .idea`

Should be gone and no longer tracked by git afterwards.